src/commonMain/kotlin/fr/acinq/lightning/EncodedNodeId.kt

```diff
@@ -16,6 +16,10 @@ sealed class EncodedNodeId {
         override fun toString(): String = if (isNode1) "<-$scid" else "$scid->"
     }
 
+    data class PhoenixId(val publicKey: PublicKey) : EncodedNodeId() {
+        override fun toString(): String = publicKey.toString()
+    }
```

Comment:
We should avoid explicitly naming `Phoenix` and use a more generic concept that can later be specified in a bLIP and implemented by other wallets (at some point we'll introduce a bLIP feature for mobile wake-up, that implies being able to read this kind of `encoded_node_id`). What about simply `WalletNodeId`? And can you similarly rename the `isPhoenix` argument to `isMobileWallet` everywhere?

```suggestion
    /**
     * Wallet nodes are not part of the public graph, and may not have channels yet. 
     * Wallet providers are usually able to contact such nodes using push notifications or similar mechanisms.
     */
    data class WalletNodeId(val publicKey: PublicKey) : EncodedNodeId() {
        override fun toString(): String = publicKey.toString()
    }
```

Reply:
Done